code/game/gamemodes/heist/tools.dm

```diff
@@ -0,0 +1,113 @@
+/**
+ * Handy-Dandy Organ Remover Doohicky
+ * SHOULD ONLY BE MAPPED ON VOX RAIDER SHIPS.
+ */
+/obj/item/weapon/organ_remover
+	name = "organics extractor"
```

Comment:
Is it supposed to be organics instead of organ? I guess it technically makes sense, but sounds weird to me.